--- ruby-1.9.3-p547/configure.in	2014-02-14 11:52:53.000000000 +0000
+++ ruby-1.9.3-p547/configure.in.new	2014-11-20 06:16:34.000000000 +0000
@@ -1009,8 +1009,7 @@
 [nextstep*], [	],
 [openstep*], [	],
 [rhapsody*], [	],
-[darwin*], [	RUBY_PREPEND_OPTION(LIBS, -lobjc)
-		RUBY_APPEND_OPTIONS(CPPFLAGS, -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE)
+[darwin*], [	RUBY_APPEND_OPTIONS(CPPFLAGS, -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE)
 		AC_MSG_CHECKING(whether Mac OS X 10.5 or later)
 		AC_TRY_CPP([#include <AvailabilityMacros.h>
 		    #if MAC_OS_X_VERSION_MAX_ALLOWED <= 1040
