--- ruby-1.9.3-p547/configure	2014-05-16 07:38:31.000000000 -0700
+++ ruby-1.9.3-p547/configure.new	2014-11-19 22:24:55.000000000 -0800
@@ -10456,15 +10456,6 @@
     	 ;; #(
   rhapsody*) :
     	 ;; #(
   darwin*) :
-    	# RUBY_PREPEND_OPTION(LIBS, -lobjc)
-	case " ${LIBS-} " in #(
-  *' -lobjc '*) :
-     ;; #(
-  '  ') :
-     LIBS="-lobjc" ;; #(
-  *) :
-     LIBS="-lobjc $LIBS" ;;
-esac
 		{ for rb_opt in -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE; do # RUBY_APPEND_OPTIONS(CPPFLAGS, -D_XOPEN_SOURCE -D_DARWIN_C_SOURCE)
 	case " ${CPPFLAGS-} " in #(
